src/main/java/org/mitre/synthea/world/concepts/RaceAndEthnicity.java

```diff
@@ -5,56 +5,20 @@
 import java.util.Map;
 
 public class RaceAndEthnicity {
-  public static final Map<String, String> LOOK_UP = createLookup();
   public static final Map<String, String> LOOK_UP_CDC_RACE = createCDCRaceLookup();
   public static final Map<String, String> LOOK_UP_CDC_ETHNICITY_CODE = createCDCEthnicityLookup();
   public static final Map<String, String> LOOK_UP_CDC_ETHNICITY_DISPLAY = 
       createCDCEthnicityDisplayLookup();
 
-  private static Map<String, String> createLookup() {
-    Map<String, String> result = new HashMap<String, String>();
-    result.put("white", "2106-3");
-    result.put("hispanic", "2135-2");
-    result.put("black", "2054-5");
-    result.put("asian", "2028-9");
-    result.put("native", "1002-5");
-    result.put("other", "2131-1");
-    result.put("irish", "2113-9");
-    result.put("italian", "2114-7");
-    result.put("english", "2110-5");
-    result.put("french", "2111-3");
-    result.put("german", "2112-1");
-    result.put("polish", "2115-4");
-    result.put("portuguese", "2131-1");
-    result.put("american", "2131-1");
-    result.put("french_canadian", "2131-1");
-    result.put("scottish", "2116-2");
-    result.put("russian", "2131-1");
-    result.put("swedish", "2131-1");
-    result.put("greek", "2131-1");
-    result.put("puerto_rican", "2180-8");
-    result.put("mexican", "2148-5");
-    result.put("central_american", "2155-0");
-    result.put("south_american", "2165-9");
-    result.put("african", "2058-6");
-    result.put("dominican", "2069-3");
-    result.put("chinese", "2034-7");
-    result.put("west_indian", "2075-0");
-    result.put("asian_indian", "2029-7");
-    result.put("american_indian", "1004-1");
-    result.put("arab", "2129-5");
-    result.put("nonhispanic", "2186-5");
-    return Collections.unmodifiableMap(result);
-  }
-
   private static Map<String, String> createCDCRaceLookup() {
     Map<String, String> result = new HashMap<String, String>();
     result.put("white", "2106-3"); // white
-    result.put("hispanic", "2106-3"); // white
+    result.put("hispanic", "2106-3"); // hispanic
     result.put("black", "2054-5"); // black
     result.put("asian", "2028-9"); // asian
     result.put("native", "1002-5"); // american indian or alaska native
-    result.put("other", "2076-8"); // native hawaiian or pacific islander
+    result.put("hawaiian", "2076-8"); // native hawaiian or pacific islander
```

Comment:
Adding a new code to `CDCRaceLookup` has a cascading effect on `Demographics.languageFromRaceAndEthnicity`. It works, in that there is a `switch` statement and individuals with the Hawaiian race code will hit the default block, which returns English as the language spoken.

This may be the best we can do for now. For other race / ethnicity combinations, we used data from the census to determine the percentage of individuals who state that they do no speak English well. For that slice of individuals, we select languages based on census data for languages spoken by race and ethnicity.

Reply:
Regarding the misalignment between demographics and the SDoH file... I'm not sure I can do anything about that, if the county files I used do not have data. I either lose SDoH data, or I lose Demographic data. The way the code should work here is that demographic data takes precedence. If the SDoH data is absent, then (unfortunately) the people in those counties do not get any of those attributes. Not ideal, but everything continues to function.

Reply:
Regarding language... good catch, I'll fix that.